libraries/botbuilder-dialogs-adaptive-runtime/src/configuration.ts

```diff
@@ -171,8 +171,9 @@ export class Configuration implements CoreConfiguration {
         try {
             return t.optional().parse(this.get(path));
         } catch (err) {
-            if (z.instanceof(z.ZodError).check(err)) {
-                err.errors.forEach((error) => (error.path = [...this.prefix, ...path, ...error.path]));
+            const zodError = z.instanceof(z.ZodError).safeParse(err);
+            if (zodError.success) {
+                zodError.data.errors.forEach((error) => (error.path = [...this.prefix, ...path, ...error.path]));
```

Comment:
I've seen this across all the PR changes, it is necessary to use `zodError.data.errors` instead of leaving the `err.errors` as is? and just use the safeParse to check it is successful or not?

Reply:
no, it can't because TS verify the type of err and throws errors because errors property "doens't exist" in unknown type 
This with all the other case, check parsed automatically the type before, but now this is not possible, for that reason I used the parsed object

Reply:
I see, i will leave this here, just in case.
![image](https://github.com/southworks/botbuilder-js/assets/62260472/f7af96fd-79c5-428a-83d4-ad675baf5b38)